### ISSUE TITLE
Loading a sound requires a path but loading an image doesn't.

### ISSUE THREAD

**Issue by fernbird** [kivy/kivy#450]
Tto load a sound, a path is required, but to load an image, that's not so.
SoundLoader.load(filename='/somedir/file.wav') 
Image(source='file.png', pos=(0,0) size=(32,32))
 i.e. SoundLoader.load(filename='file.wav') doesn't work

I and probably most other python programmers would assume the working directory is whatever directory main.py is in. so loading file.wav should look for file.wav in the same dir as main.py, and 'subdir/file.wav' would be a subdirectory of the directory main.py is in.

In any case, whatever method is used for loading resources, it should at least be consistent.

tl;dr: use resource_find for SoundLoader?
